pages/components/cards.js

```diff
@@ -209,6 +224,24 @@ export default function Cards({ posts, setPosts }) {
                   <button type="submit">Submit</button>
                 </form>
               )}
+              {post.isEditing && (
+                <form onSubmit={(event) => handleSaveComment(event, post.id)}>
+                  <textarea
+                    placeholder="Edit your comment"
+                    value={post.editedContent}
+                    onChange={(event) =>
+                      setPosts(
+                        posts.map((p) =>
+                          p.id === post.id
+                            ? { ...p, editedContent: event.target.value }
```

Comment:
why is this key called editedContent and not just content?

Reply:
because in the json file it is called content. so it is easier for the eye of the viewer